pkg/handler/iam_handler.go

```diff
@@ -181,8 +178,7 @@ func updatePolicy(p *iampb.Policy, bs []*v1alpha1.Binding, expiry time.Time) err
 		// Skip expired bindings.
 		expired, err := expired(cb.Condition.Expression)
 		if err != nil {
-			// Return error immediately since we don't expect this to fail.
```

Comment:
No op - now I realized this is the only error it will return

Reply:
Yeah, but it was confusing if it was logged there.